From 1314130c895fb168f48104d25ebfe2fff20e3711 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Thu, 11 Jun 2015 21:15:27 -0700
Subject: [PATCH 49/67] SoftVideoDecoderOMXComponent: max-size can be less than
 size

A component can be configured for size that is greater than the
max size. In such case, configure for the larger of these.
---
 media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp b/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
index 935dc340b..9d1798622 100644
--- a/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
+++ b/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
@@ -185,11 +185,11 @@ void SoftVideoDecoderOMXComponent::updatePortDefinitions(bool updateCrop, bool u
 
 
 uint32_t SoftVideoDecoderOMXComponent::outputBufferWidth() {
-    return max(mIsAdaptive ? mAdaptiveMaxWidth : 0, mWidth);
+    return mIsAdaptive ? mAdaptiveMaxWidth : mWidth;
 }
 
 uint32_t SoftVideoDecoderOMXComponent::outputBufferHeight() {
-    return max(mIsAdaptive ? mAdaptiveMaxHeight : 0, mHeight);
+    return mIsAdaptive ? mAdaptiveMaxHeight : mHeight;
 }
 
 void SoftVideoDecoderOMXComponent::handlePortSettingsChange(
-- 
2.11.0

